src/util/textInput.js

```diff
@@ -33,5 +33,8 @@ const onKeyCode = curry((modifier, keyCode, callback, event) =>
 // use like: <input type='text' onKeyDown={onEnter(callback)}/>
 export const onEnter = onKeyCode(null, keyMap.ENTER)
 export const onCmdEnter = onKeyCode('metaKey', keyMap.ENTER)
+export const onCtrlEnter = onKeyCode('ctrlKey', keyMap.ENTER)
+export const onCmdOrCtrlEnter = (callback, event) =>
```

Comment:
Ramda appears to have "[either](http://ramdajs.com/0.22.1/docs/#either)" but i don't think there's anything in lodash/fp for this.